src/module/SynchrotronRadiation.cpp

```diff
@@ -8,21 +8,23 @@
 
 namespace crpropa {
 
-SynchrotronRadiation::SynchrotronRadiation(ref_ptr<MagneticField> field, bool havePhotons, double limit) {
-	Brms = 0.;
+SynchrotronRadiation::SynchrotronRadiation(ref_ptr<MagneticField> field, bool havePhotons, double thinning, int nSamples, double limit) {
 	setField(field);
+	setBrms(0);
 	initSpectrum();
-	this->havePhotons = havePhotons;
-	this->limit = limit;
-	secondaryThreshold = 1e7 * eV;
+	setHavePhotons(havePhotons);
+	setLimit(limit);
+	setSecondaryThreshold(1e6 * eV);
```

Comment:
What was the reason for changing the default value of the threshold?

Reply:
This choice was arbitrary. The previous value had no physical meaning, neither does this. 
It would make sense to choose either 1 GeV, which is roughly the minimum energy that we treat with CRPropa, or 1 keV, since X-ray energies are typical for synchrotron emission.

Reply:
In that case I would prefer to keep the old value. Just to prevent any possible confusion.